internal/view/svc.go

```diff
@@ -60,6 +60,10 @@ func (s *Service) showPods(a *App, _ ui.Tabular, gvr, path string) {
 		return
 	}
 
+	if svc.Spec.Type == v1.ServiceTypeExternalName {
+		a.Flash().Warnf("No <enter> view for %s services. They do not target pods", v1.ServiceTypeExternalName)
```

Comment:
Please rework "No <enter>..." -> "No pod view available. Service %s is an external service."??

Reply:
Thank you Daniel!